modules/core/src/ocl.cpp

```diff
@@ -1330,7 +1330,7 @@ class AmdFftHelper
     {
         try
         {
-//            clAmdFftTeardown();
+            clAmdFftTeardown();
```

Comment:
In general, we can't do that, because OpenCV library is not a final application.
Other components of user application can still use clFFT even if OpenCV library is unloaded.

Also there are issues may happen with Win32 ExitProcess and OpenCL runtime (it may just crash / hang on cleanup calls).

Reply:
`clAmdBlasTeardown` is already being called in a similar way:

https://github.com/opencv/opencv/blob/52d96faa7557006f04b95bd38979586d05d9c314/modules/core/src/ocl.cpp#L1255-L1262

What do you suggest?  It seems to me that OpenCV should either tear down both clBLAS and clFFT _or_ it should tear down neither.

Reply:
I believe we should disable both.
Also it makes sense to add comment to avoid future similar changes.

Reply:
Good, I will do that.